Airports/EGFF/Ownership.txt

```diff
@@ -2,13 +2,13 @@ SECTOR:Only FFTWR 1:0:0
 OWNER:FFR:FFT
 
 SECTOR:Only FFTWR 2:0:0
-OWNER:FFR:LW:LWN:LWC:LWS:L:LM:O
+OWNER:FFR:LWH:LWX:LW:LWN:LWC:LWS:L:LM:O
```

Comment:
Just checking you meant to change the Cardiff top-down order to be S6 (H->X) and leave the Bristol top-down order as S23 (R->B->P). I suppose it spreads the workload but results in a lot of weirdness when 23 is on but no GD (i.e. 5/8 to 6 to 23 for Bristol inbounds).

Reply:
I think the Cardiff top-down order should be R->B->P